README.md

```diff
@@ -55,6 +55,31 @@ pokerEvaluator.evalHand(['As', 'Ac', 'Qs']);
 //  handName: 'one pair' }
 ```
 
+Alternatively using the original API (less recommended):
+
+```js
+const PokerEvaluator = require('poker-evaluator');
+
+PokerEvaluator.evalHand(['As', 'Ks', 'Qs', 'Js', 'Ts', '3c', '5h']);
+//{ handType: 9,
+//  handRank: 10,
+//  value: 36874,
+//  handName: 'straight flush' }
+
+```
+
+With numbers: 
+```js
+const PokerEvaluator = require('poker-evaluator');
+
+PokerEvaluator.evalHand([17, 22, 27, 32, 33]);
+//{ handType: 5,
+//  handRank: 6,
+//  value: 20486,
+//  handName: 'straight' }
+
+```
```

Comment:
Happy to accept another PR by you for this example of number usage, otherwise I'll just copy and paste this :)

Reply:
You can just copy paste it. I don't think it's worth a commit + pr for a 5 line example. Thanks though.

> If we do that and just export the evalHand function it keeps the consistency of the original api in CommonJS but means would have to do `const PokerEvaluator = require('poker-evaluator');` if we are consuming this in a typescript file.

Could you not use an ES6 style import and do
```js
import PokerEvaluator from 'poker-evaluator';

Pokerevaluator.evalhand(["As", "Ks", "Qs", "Js", "Ts", "3c", "5h"]);
```
I think this is equivalent to the CommonJs require, but I may be mistaken.
https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Statements/import

Reply:
Ah actually `import * as PokerEvaluator from 'poker-evaluator';` works. Will update the readme